packages/language-server/src/server.ts

```diff
@@ -142,9 +143,12 @@ export function startServer(options?: LSPOptions): void {
 
   // function getDocumentSettings(resource: string): Thenable<LSPSettings> {
   //   if (!hasConfigurationCapability) {
-  //     connection.console.info(`Using default prisma-fmt binary path.`)
+  //     connection.console.info(
+  //       `hasConfigurationCapability === false. Defaults will be used.`,
```

Comment:
I don't understand this edit — was it intentional?

Reply:
I modified it but in the end the code was not needed